docs/upgrade-to-1.0.md

```diff
@@ -42,6 +42,19 @@ _*backups created prior to v0.10 stored snapshot metadata in the `status.volumeB
     velero backup-location get
     ```
 
+1. Fetch your bucket credentials and make them accessible to Velero
+
+    ```bash
+    # AWS
+    AWS_SHARED_CREDENTIALS_FILE=$(kubectl get secrets/<YOUR_SECRET_NAME> -n velero -o jsonpath="{.data.cloud}" | base64 -d)
```

Comment:
this is going to dump the content of the secret into the env var, right? I'm not sure that works - we may need to write the data into a local file, then set the env var to have the path of the file

Reply:
Yeah, you're right. For AWS/Google we'll need to make (and clean up) a temp file for the operation.